haralyzer/assets.py

```diff
@@ -407,7 +407,7 @@ def time_to_first_byte(self):
         initial_entry = self.entries[0]
         ttfb = 0
         for k, v in iteritems(initial_entry['timings']):
-            if k != 'receive':
+            if k != 'receive' and isinstance(v, int)
```

Comment:
Should we maybe be casting to an integer instead? I'm guessing that what happened here is that the HAR file you used has a string value (something like "50" instead of 50) instead of an int value in the JSON. Can you confirm?